### DISCUSSION TITLE
v2.15.0

### DISCUSSION THREAD

**Discussion by sojan-official** [chatwoot/chatwoot#6677] [Releases & Announcements]
## Changelog

- Conversation Heatmap reports
- New Search experience
- RTL language support in Chatwoot Dashboard
- Audit Log APIs (EE edition)
- Support Rich messages in pre-chat form
- Support for WhatsApp messages from older Brazilian phone numbers
- Push notification support for safari
- Integration with Logrocket
- Numerous bug fixes and enhancements

###
Thanks to @giautm, @afonsolage, @gitstart, @raph941, @phunguyenmurcul, @clairton, @BikashSah999, @mani-zur, @CristianDuta, @jacsonsantospht  for the contributions

<hr /><em>This discussion was created from the release <a href='https://github.com/chatwoot/chatwoot/releases/tag/v2.15.0'>v2.15.0</a>.</em>

**Comment by sojan-official** [comment]
## 2.15.0 Dev Changelog

### API Endpoints

#### Additions
- `/api/v1/accounts/{account.id}/search` GET[/contacts, /conversations , /messages]
     - https://github.com/chatwoot/chatwoot/pull/6564
     - Ability to search across messages, conversations and contacts using a query term

#### Deletions
- none

---

### Other Changes

- `/api/v1/accounts/{account.id}/audit_logs` [GET]
  - https://github.com/chatwoot/chatwoot/pull/6434
  -  Fetch audit logs for an account. Only available in enterprise edition.

**Comment by mmojadad** [comment]
Hi
When RTL language support in Chatwoot Widget will be available?
